package.json

```diff
@@ -35,6 +35,7 @@
     "benderjs-promise": "^0.1.0",
     "benderjs-sinon": "^0.3.0",
     "chai": "^3.4.0",
+    "ckeditor5-dev-utils": "ckeditor/ckeditor5-dev-utils",
```

Comment:
This was intentional. The linting package will be required by all other packages anyway and using links everywhere makes no sense. So I think we should use a normal workflow here with releases and publishing. I even wrote in https://github.com/ckeditor/ckeditor5/issues/215#issuecomment-234923178 that we need to update `package.json` after we finish refactoring to use versioned packages.